kinds/dashboard/dashboard_kind.cue

```diff
@@ -435,6 +435,8 @@ lineage: schemas: [{
 			index?: int32
 		} @cuetsy(kind="interface")
 
+		#DataFramesSource: "series" | "annotations" @cuetsy(kind="enum",memberNames="Series|Annotations")
```

Comment:
we should probably use the existing [DataTopic](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/query.ts#L19) since that is the field that is filled for annotations.

Currently everythign without topic=Annotations is a series, but we do not have an explicit id for it